test/controllers/organizations/staff/external_form_upload_controller_test.rb

```diff
@@ -0,0 +1,21 @@
+require "test_helper"
+
+class Organizations::Staff::ExternalFormUploadControllerTest < ActionDispatch::IntegrationTest
+  setup do
```

Comment:
Ok these tests look good. 
Any reason we set up one adopter and one staff? We should ideally set up two adopters and then assert that one of them will get updated form answers, and the other one does not. Does that make sense?

Reply:
Yes it does 👍🏽